test/Microsoft.IdentityModel.Tokens.Tests/SignatureProviderTests.cs

```diff
@@ -939,14 +940,13 @@ private void SymmetricProviderDispose(string testId, SecurityKey securityKey, st
 
             return theoryData;
         }
-#if NET_CORE
-        // Excluding OSX as SignatureTampering test is slow on OSX (~6 minutes)
-        // especially tests with IDs RS256 and ES256
-        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)]
-#endif
+
         [Theory, MemberData(nameof(SignatureTheoryData))]
         public void SignatureTampering(SignatureProviderTheoryData theoryData)
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                throw new PlatformNotSupportedException("OSX is excluded as the SignatureTampering test is slow (~6 minutes).");
```

Comment:
this doesn't look right, before OSX platforms would skip, now they will fail on this test?